src/common/colorspaces.c

```diff
@@ -2398,9 +2398,7 @@ void dt_colorspaces_cygm_apply_coeffs_to_rgb(float *out,
         RGB_to_RGB_WB[a][b] += CAM_to_RGB_WB[a][c] * RGB_to_CAM[c][b];
     }
 
-#ifdef _OPENMP
-#pragma omp parallel for default(none) shared(in, out, num, RGB_to_RGB_WB) schedule(static)
-#endif
+  DT_OMP_FOR(shared(in, out, num, RGB_to_RGB_WB))
```

Comment:
This is one of those cases I've previously mentioned where `shared` variables would be better as `firstprivate`.  `num` is read-only and a simple integer, and the other three are pointer variables which aren't modified and where there is no write contention on the pointed-at buffer because the portion of the buffer that is modified for `out` is disjoint among threads.

Reply:
This patch set is meant to be purely cosmetic, i.e. to improve readability/convenience. As such it intentionally tries not to make any functional changes that could introduce regressions. Instances where that might happen (like mixed `for simd` / `schedule(simd:static)` cases) are left untouched. Most of the instances where all variables were marked as shared should be in the last commit, so you could decide to leave that one out when/if you decide to push this upstream and then review the remaining `#pragma parallel for` at your leisure.

Reply:
Understood.  Just pointing out that this is something to be looked at *eventually*.
Easier to convert everything now and then review all the shared() clauses at (as you say) our leisure.